Foundation/NSNotification.swift

```diff
@@ -187,8 +187,13 @@ open class NotificationCenter: NSObject {
             self._observers = _observers.filterOutObserver(observer, name: aName, object: object)
         })
     }
-    
+
+    @available(swift, obsoleted: 4.0)
```

Comment:
Although you can have two `@available` attributes, you can also combine them for different platforms:

https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Attributes.html

So `@available(*,obsoleted:4.0,renamed:"addObserver(forName:object:queue:using:)")` will work in this case.